src/app/workflow/versions/versions.component.ts

```diff
@@ -245,4 +247,20 @@ export class VersionsWorkflowComponent extends Versions implements OnInit, OnCha
   trackBy(index: number, item: WorkflowVersion) {
     return item.id;
   }
+
+  showExecutionMetricsIcon(version: WorkflowVersion) {
+    if (version.metricsByPlatform !== null) {
```

Comment:
Can it be `undefined`? I suspect the API always returns something, so probably not, but just double-checking. If unsure, it could be safer to do `if (version.metricsByPlatform) {`